File: app/src/main/java/com/sami/rippel/views/GlideConfiguration.java
package com.sami.rippel.views;

import android.content.Context;

import com.bumptech.glide.GlideBuilder;
import com.bumptech.glide.load.DecodeFormat;
import com.bumptech.glide.module.AppGlideModule;
import com.bumptech.glide.request.RequestOptions;

@com.bumptech.glide.annotation.GlideModule
public class GlideConfiguration extends AppGlideModule {

    @Override
    public void applyOptions(Context mContext, GlideBuilder mBuilder) {
        mBuilder.setDefaultRequestOptions(new RequestOptions().format(DecodeFormat.PREFER_ARGB_8888));
    }

    @Override
    public boolean isManifestParsingEnabled() {
        return false;
    }
}

File: app/src/main/java/com/sami/rippel/utils/GlideConfiguration.kt
package com.sami.rippel.utils

import android.content.Context
import com.bumptech.glide.GlideBuilder
import com.bumptech.glide.annotation.GlideModule
import com.bumptech.glide.load.DecodeFormat
import com.bumptech.glide.module.AppGlideModule
import com.bumptech.glide.request.RequestOptions

@GlideModule
class GlideConfiguration : AppGlideModule() {

    override fun applyOptions(
        context: Context,
        mBuilder: GlideBuilder
    ) {
        mBuilder.setDefaultRequestOptions(RequestOptions().format(DecodeFormat.PREFER_ARGB_8888))
    }

    override fun isManifestParsingEnabled() = false
}